omod/src/main/webapp/pages/results.gsp

```diff
@@ -143,8 +204,9 @@ def prettify = { "$it".replaceAll("null", "").replaceAll("\\s*,\\s", " ") }
 <table id="list-results" cellspacing="0" cellpadding="2" width="100%" class="dataTable">
     <thead>
     <tr>
-        <th style="display: none"></th>
-        <th>${ui.message("dataintegrity.rule.name")}</th>
+        <th class="hidden"></th>
+        <th style="width:15%">Identifier</th>
```

Comment:
Use classes instead of hardcoding style tags - this will become important later when responsive behavior is added